var/spack/repos/builtin/packages/callpath/package.py

```diff
@@ -46,7 +46,9 @@ def install(self, spec, prefix):
         # TODO: offer options for the walker used.
         cmake_args = std_cmake_args
         if spec.satisfies("^dyninst@9.3.0:"):
-            cmake_args.append("-DCMAKE_CXX_FLAGS='-std=c++11 -fpermissive'")
+            std_flag = self.compiler.cxx11_flag
+            cmake_args.append("-DCMAKE_CXX_FLAGS='{0} -fpermissive'".format(
+                    std_flag))
```

Comment:
I was wondering if we could use [CMAKE_CXX_STANDARD](https://cmake.org/cmake/help/v3.1/variable/CMAKE_CXX_STANDARD.html#variable:CMAKE_CXX_STANDARD) instead of setting the flags manually. It requires `cmake@3.0.0:` I think.